tests/api/test_security.py

```diff
@@ -0,0 +1,81 @@
+import requests
+import unittest
+import os
+
+
+class Testsecurity(object):
```

Comment:
This class should be called TestPOST or alike, depending on which methods are tested.